.github/ci-test-each-commit-exec.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# Copyright (c) The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+import subprocess
+import sys
+import shlex
+
+
+def run(cmd, **kwargs):
+    print("+ " + shlex.join(cmd), flush=True)
```

Comment:
Could add file=sys.stderr if want to emulate bash tracing (not necessarily better though)